base/src/main/groovy/org/asciidoctor/gradle/base/AbstractAsciidoctorBaseTask.groovy

```diff
@@ -241,13 +241,13 @@ abstract class AbstractAsciidoctorBaseTask extends DefaultTask {
      * @param includePatterns ANT-style patterns for sources to include
      */
     void sources(String... includePatterns) {
-        new Action<PatternSet>() {
+        sources(new Action<PatternSet>() {
```

Comment:
:man_facepalming: 

Please add you name to the author list for this class.

Reply:
Done

Reply:
btw. the same issue remains in master branch. https://github.com/asciidoctor/asciidoctor-gradle-plugin/blob/63c508caffb7fb6a3331732f3c7c8e00005c44fd/asciidoctor-gradle-jvm/src/main/groovy/org/asciidoctor/gradle/jvm/AbstractAsciidoctorTask.groovy#L180-L188

Reply:
Final release of 3.x will go out in January, so then `development-3.x` will effectively become `master`. I prefer fixes coming into `development-3.x` at this point.